### PR TITLE
Added governance exercise

### PR DESCRIPTION
**Pull request by Ainali**
Put together the first version of the exercise into a package as result of #437. 

-----
[View rendered activities/codebase-stewardship/governance-exercise.md](https://github.com/publiccodenet/about/blob/add-governance-exercise/activities/codebase-stewardship/governance-exercise.md)
[View rendered activities/codebase-stewardship/product-assets-for-early-incubation.md](https://github.com/publiccodenet/about/blob/add-governance-exercise/activities/codebase-stewardship/product-assets-for-early-incubation.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Great PR and very happy to see this here.

I'd like this to live in a separate 'activity' and have a path like `activities/workshops/governance/` or something.

And as a separate question, why are the assets in PDFs and not some form of source file such as an Inkscape SVG? Posing this mostly from a source control perspective as these kinds of files are often hard to manage.

**Comment by Ainali** [comment]
I agree that PDFs are harder to handle, I made that choice mostly because they are several pages and SVGs don't handle multiple pages (as far as I know). However, Inkscape can easily open a single page of the PDF and edit it so the obstacle is not that big.

**Comment by bvhme** [comment]
That is fair, I just didn't know that it was also fairly easy to edit those PDFs again.

Be aware of the merge conflict.

**Comment by clausmullie** [comment]
Super cool!

File: activities/workshops/governance-exercise.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  - Reason: to get their brains started and see that the process is both complex and probably incomplete.
+
+- An ideal situation
+
+  - Reason: If the participants get to design a new process after describing what they have, they will be part of a process of change. Since they themselves have identified a gap to where they want to be they have internalized that change is coming.
+
+- Discussion
+
+  - Reason: By discussing what they came up with they will realize what went through the different participants heads.
```
Comment (bvhme) [inline activities/workshops/governance-exercise.md]: This is invalid Markdown and might yield different results on different renders. 

I suggest using lists like:

```markdown
* A
  * b
* C
  * d
* E
  * f
```

however the following might be more fitting:

```markdown
1. A: b
2. C: d
3. E: f
```

or transforming it into headings with content.

**Comment by Ainali** [comment]
There is an index (although very short) for workshops (format borrowed from trainings)

**Comment by bvhme** [comment]
@Ainali in the 'Files changed' and when I browse the files I find that the `index.md` is empty, perhaps you have not committed the content yet.

Thanks :)

**Comment by ericherman** [comment]
I didn't notice the _absence_ of the index content! good catch @bvhme !

**Comment by Ainali** [comment]
That was a strange omission from me. Now there is content in the index!

**Comment by Ainali** [comment]
Follow up will be made in #457.

**Comment by bvhme** [comment]
Awesome, thanks for the hard work!

**Comment by Ainali** [comment]
Fixes #437

**Pull request merged**